recipe-client-addon/test/browser/browser_about_studies.js

```diff
@@ -64,22 +64,27 @@ compose_task(
   }),
   withAboutStudies,
   async function testUpdatePreferencesOldOrganization(browser) {
-    browser.contentDocument.getElementById("shield-studies-update-preferences").click();
-    await BrowserTestUtils.waitForLocationChange(gBrowser);
-    await BrowserTestUtils.waitForEvent(browser.contentWindow, "load");
+    // We have to use gBrowser instead of browser in most spots since we're
+    // dealing with a new tab outside of the about:studies tab.
+    const tab = await BrowserTestUtils.switchTab(gBrowser, () => {
+      browser.contentDocument.getElementById("shield-studies-update-preferences").click();
```

Comment:
This will use a CPOW (cross process wrapper) now that this is really running in the child process. We're aiming to get rid of those.

Instead, you can use:
```js
ContentTask.spawn(browser, null, () => {
  content.document.getElementById("shield-studies-update-preferences").click();
});
```
which will run that function in the content process (as a new frame script, essentially).